### PR TITLE
Clear BTC wallet data after wallet switches

### LINKED ISSUE #187: Clear BTC wallet data after wallet switches
When switching from SafeCard <-> Lattice it doesn't look like the wallet data is automatically cleared. The user needs to hit the refresh button to load the wallet data for the new interface. This should not be the case.

### PR DESCRIPTION
**Pull request by douglance**
Adds a poll to refresh the user's wallet every 8 minutes or so. 

While refreshing, the entire UI is taken over by the refresh loader. I'm not sure if this user experience is worth the refreshing action. 

We can't refresh without the UI take-over, because the user could be in the middle of an action and swapping out the data out from under them could lead to them performing undesirable actions.  

Fixes #187

### CODE REVIEW

**Review by alex-miller-0 — CHANGES_REQUESTED** [review]
The refresh is a good idea but the underlying issue is related to wallet state data not updating if the interface does.

You can press the Lattice1/SafeCard button on the header to switch interface, but this does not update the wallet data. To do that, you need to press refresh (different button) on the wallet screen itself.

**Comment by douglance** [comment]
I see. I didn't have a good understanding of the issue. 

I swapped the refresh from the overall wallet to just the Btc refresh, and throttled it a bit on the data's last refresh, so the UX isn't impacted as much. 

I didn't get to test the change as much as I wanted, as my requests were throttled by one of our services we use to fetch the data. I'll keep testing tomorrow.

**Comment by alex-miller-0** [comment]
@douglance The fix to the issue itself is probably a one liner: https://github.com/GridPlus/lattice-manager/issues/187#issuecomment-1005186844

**Comment by douglance** [comment]
@alex-miller-0 I made that change and revised the history. I can't test much right now, because I'm still rate limited.

**Comment by alex-miller-0** [comment]
@douglance looks like you removed the refresh logic all together? I think that was fine (although not addressing the linked issue) if you want to add it back in.

Not sure how you could be rate limited but you could try switching to a VPN. This should not pile up many requests so not sure what could be going on

**Comment by douglance** [comment]
> @douglance looks like you removed the refresh logic all together? I think that was fine (although not addressing the linked issue) if you want to add it back in.

I wasn't happy with the user experience of the entire app being taken over by a loading indicator. We can do a better UX when we get a chance to update how data is handled in the app.

File: src/components/main.js [hunk 1]
```diff
@@ -435,6 +435,7 @@ class Main extends React.Component {
         return this.setError({ msg: err, cb: this.refreshWallets })
       this.setError();
     })
+    this.fetchBtcData()
```
Comment (alex-miller-0) [inline src/components/main.js:438]: Shouldn't this be inside the callback? Like right after `this.setError()`?

Comment (douglance) [reply src/components/main.js]: Yep. It should.

I'm still having issues getting `blockchain.info` requests to go through. It's sending `429` status code responses every time. I'm looking into it.

Comment (douglance) [reply src/components/main.js]: d801487

Comment (alex-miller-0) [reply src/components/main.js]: That's super weird... okay this change is good

**Review by alex-miller-0 — APPROVED** [review]

**Pull request merged**